src/audio/pipeline.c

```diff
@@ -1263,7 +1263,7 @@ void pipeline_schedule_copy(struct pipeline *p, uint64_t start)
  * buffers prior to trigger start. */
 void pipeline_schedule_copy_idle(struct pipeline *p)
 {
-	schedule_task_idle(&p->pipe_task, p->ipc_pipe.deadline);
+	schedule_task(&p->pipe_task, 0, p->ipc_pipe.deadline);
 }
```

Comment:
So it should not be named pipeline_schedule_copy_idle()?
And this simple code may impact a lot. Please do a stress test, especially with encoding/decoding and other tasks together. I'm not confident on the side effect of this code.